src/javascript/binary/base.js

```diff
@@ -222,3 +222,9 @@ if (!/backoffice/.test(document.URL)) { // exclude BO
         LocalStore.set('active_loginid', match);
     });
 }
+
+
+//this method is to prevent caching on sensitive data
+pjax_config_page("new_account/real", function(){
+    $("#secretanswer").attr("autocomplete", "off");
```

Comment:
One more reason not to generate complex html with server-side code ;)

It is fine as it is then. No concerns.